packages/strapi-design-system/src/MainNav/NavLink.js

```diff
@@ -62,13 +62,12 @@ const MainNavRow = styled(Flex)`
 `;
 
 const CustomBadge = styled(Badge)`
-  ${({ condensed }) =>
+  ${({ theme, condensed }) =>
     condensed &&
     `
 	  position: absolute;
-    transform: translate(35%, -50%);
-    top: 0;
-    right: 0;
+    top: -${theme.spaces[3]};
+    right:  -${theme.spaces[1]};
```

Comment:
What are these numbers based on? Is that half the height/ width of the Badge?

Reply:
to be honest, they are mainly based on the visual aspect - checked and approved with @maevalienard - do you think we need another system?

Reply:
No, but I think this deserves a comment :)